src/Nethermind/Nethermind.Core2.Abstractions/Types/CommitteeIndex.cs

```diff
@@ -22,21 +22,14 @@ namespace Nethermind.Core2.Types
     [DebuggerDisplay("{Number}")]
     public struct CommitteeIndex : IEquatable<CommitteeIndex>, IComparable<CommitteeIndex>
     {
-        public const int SszLength = sizeof(ulong);
-
         public CommitteeIndex(ulong number)
         {
             Number = number;
         }
         
         public ulong Number { get; }
-        
-        public bool Equals(ValidatorIndex other)
-        {
-            return Number == other.Number;
-        }
-        
-        public static CommitteeIndex None => new CommitteeIndex(ulong.MaxValue);
+
+        public static CommitteeIndex None => new CommitteeIndex(ulong.MaxValue - 1);
```

Comment:
is this as in spec or should we use nullables were None is required?

Reply:
Using a nullable would probably be more in line with the spec, which actually specifies None.

ulong.MaxValue is in the spec as FarFutureEpoch, for Slot; which is why using MaxValue as special value for none has issues.

I'll add it as a work item to the backlog.